lib/auth/auth.go

```diff
@@ -474,6 +480,13 @@ type Server struct {
 	loadAllCAs bool
 }
 
+// RegisterSAMLService registers ss as the SAMLService that provides the SAML
+// connector implementation. If a SAMLService has already been registered, this
+// will override the previous registration.
+func (a *Server) RegisterSAMLService(ss SAMLService) {
```

Comment:
Nit: I think `SetSAMLService` would be more in line with how we usually name methods like this.

Reply:
Done: 22d628c80582a5f099e779a734062c5eb3575295